extensions/bear/src/db-queries.ts

```diff
@@ -205,6 +205,8 @@ FROM
   ZSFNOTE AS notes
   LEFT OUTER JOIN Z_7TAGS AS notes_to_tags ON notes.Z_PK = notes_to_tags.Z_7NOTES
   LEFT OUTER JOIN ZSFNOTETAG AS tags ON notes_to_tags.Z_14TAGS = tags.Z_PK 
+-- Ignore tags for trashed and archived notes
+WHERE notes.ZARCHIVED = 0 AND notes.ZTRASHED = 0
```

Comment:
Some thoughts I had:

1. It doesn't make sense to provide a preference to show/hide tags if we are not providing preference to hide archived and trashed notes.
2. A note with a tag will not be empty and tags are not encrypted. So this condition is not required:

https://github.com/raycast/extensions/blob/dedf880749545861a71cb0083030441f4eae5c3e/extensions/bear/src/db-queries.ts#L28-L31